### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by Rafi594**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -3,9 +3,10 @@
     "id": "collabora",
     "packaging_format": 1,
     "description": {
-        "en": "Collabora online package for YunoHost.",
-        "fr": "Collabora online pour YunoHost."
+        "en": "Collabora online.",
```
Comment (alexAubin) [inline manifest.json]:
```suggestion
        "en": "A LibreOffice-based online office suite with collaborative editing",
```

File: manifest.json [hunk 2]
```diff
@@ -3,9 +3,10 @@
...
+        "fr": "Collabora online."
```
Comment (alexAubin) [inline manifest.json]:
```suggestion
        "fr": "Une suite office en ligne et collaborative, basée sur LibreOffice"
```

**Comment by Rafi594** [comment]
Ready to merge for me

**Comment by alexAubin** [comment]
Soooo anybody feel like clicking the button ? :P

**Pull request merged**